### PR TITLE
Update us.m3u group-title

### PR DESCRIPTION
**Pull request by Tribalize**
Update group-title, tvg-language

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="KOTR-LD.us" tvg-name="11 MyTV (KOTR-LD)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/en/b/bd/Kotrlp.jpg" group-title="Local",11 MyTV (KOTR-LD) (480i)
 https://cdnapisec.kaltura.com/p/2630451/sp/2630451/playManifest/entryId/1_hq4vlk40/deliveryProfileId/672/protocol/https/format/applehttp/a.m3u8
-#EXTINF:-1 tvg-id="13thStreet.us" tvg-name="13th Street (Poland)" tvg-country="PL" tvg-language="Polish;English" tvg-logo="" group-title="",13th Street (Poland)
+#EXTINF:-1 tvg-id="13thStreet.us" tvg-name="13th Street (Poland)" tvg-country="PL" tvg-language="Polish" tvg-logo="" group-title="Entertainment",13th Street (Poland)
```
Comment (LaneSh4d0w) [inline channels/us.m3u:4]: Not English ? There's both languages available on this one.

File: channels/us.m3u [hunk 2]
```diff
@@ -3691,9 +3687,9 @@ http://wpso.com:1936/hls/wzra.m3u8
...
 https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/tracks-v1a1/index.m3u8
-#EXTINF:-1 tvg-id="ZooMoo.us" tvg-name="Zoo Moo" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/1wYnA2i.jpg" group-title="",Zoo Moo (720p)
+#EXTINF:-1 tvg-id="ZooMoo.us" tvg-name="Zoo Moo" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/1wYnA2i.jpg" group-title="Kids",Zoo Moo (720p)
 https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-name="Канал Disney" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Kids",Канал Disney (576p)
 http://188.40.68.167/russia/disney/playlist.m3u8
-#EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
+#EXTINF:-1 tvg-id="CheddarNews.us" tvg-name="Cheddar News" tvg-country="US" tvg-language="English" tvg-logo="https://miro.medium.com/max/2160/1*nrRFfv8W3RuFZQpskTIH3A.jpeg" group-title="News",Cheddar News (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3695]: Not sure about that one. Is it the English Cheddar News ?

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (Tribalize) [reply channels/us.m3u:4]: technically yes. english is in the background but the overdub of polish you can listen.

File: channels/us.m3u (hunk 2 above)
Comment (Tribalize) [reply channels/us.m3u:3695]: yes. tested link and that is whats pulled

File: channels/us.m3u (hunk 1 above)
Comment (Tribalize) [reply channels/us.m3u:4]: tested again. Not all shows are overdubbed. I will revert back

File: channels/us.m3u (hunk 2 above)
Comment (Tribalize) [reply channels/us.m3u:3695]: Tested again. it is cheddar news

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u]: If you're using VLC, you got the choice of using both undubbed / dubbed versions.

File: channels/us.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:3695]: Doesn't even seem to work for me. I'm only getting advertisments for now.

Comment (Tribalize) [reply channels/us.m3u:3695]: tested again to triple check. comes up the same

**Review by BellezaEmporium — APPROVED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (Tribalize) [reply channels/us.m3u]: i normally test vlc , kodi, and streamtest.in. i have reverted this change and updated pr

File: channels/us.m3u (hunk 2 above)
Comment (Tribalize) [reply channels/us.m3u:3695]: glad it wasnt just me

**Pull request merged**